.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

```suggestion
      interval: "monthly"
```

Can you change this to monthly? This project is both does not have a lot of development speed, nor does it need a lot of attention.

I really don't want a weekly pull request submitted by dependabot that causes more work than it actually solves (because the version bump is commonly very trivial anyway, but the work to verify that everything still works as expected isn't done by dependabot). Limiting this to a monthly cadence would make this much more manageable.